src/OpenTelemetry/Metrics/Implementation/Metric.cs

```diff
@@ -23,15 +23,40 @@
 
 namespace OpenTelemetry.Metrics.Implementation
 {
+    /// <summary>
+    /// This class need to evolve to what's referred to as Export Record in the spec.
+    /// An exporter-independent in-memory representation combining the metric instrument, the LabelSet for export, and the associated aggregate state.
+    /// </summary>
     public class Metric
     {
-        public Metric(string metricNamespace, string metricName, string desc, IEnumerable<KeyValuePair<string, string>> labels, long value)
+        public Metric(string metricNamespace,
+            string metricName,
+            string desc,
+            InstrumentKind kind,
```

Comment:
why does metric has an instrument type? Should it be "aggregation type" instead?